lib/xlsx_reader/parsers/worksheet_parser.ex

```diff
@@ -250,6 +272,21 @@ defmodule XlsxReader.Parsers.WorksheetParser do
     Utils.map_attributes(attributes, @cell_attributes_mapping)
   end
 
+  defp format_cell_data(state) do
+    value = convert_current_cell_value(state)
+
+    expanded_cell_data = %{
+      value: value,
+      formula: state.formula,
+      cell_ref: state.cell_ref
+    }
+
```

Comment:
@xavier I think having a more general "expanded" format is nicer than the return_formula I had before. We could add cell_style, or other attributes here in the future. It also makes it such that the return type is either a primative, or a Map and not a mixture of both (regardless of whether or not there is a formula present)

Reply:
good call @xavier, I made the update